src/iap/shared/interceptors/http-error.interceptor.ts

```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { HttpErrorResponse, HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';
+import { catchError, Observable, throwError } from 'rxjs';
```

Comment:
:point_right: :point_left: 
```suggestion
import { HttpErrorResponse, HttpEvent, HttpHandler, HttpInterceptor, HttpRequest } from '@angular/common/http';

import { catchError, Observable, throwError } from 'rxjs';
```